### PR TITLE
Added LinkedIn links to contact cards

### PR DESCRIPTION
**Pull request by bricct**
Wraps the contact cards on the about page in style-less containers that reroute to respective linkedin pages.

### CODE REVIEW

File: src/routes/About.tsx [hunk 1]
```diff
@@ -8,60 +8,64 @@ import AwsImage from "../assets/images/aws.png";
 import ReactImage from "../assets/images/react.png";
 import FlaskImage from "../assets/images/flask.png";
 import SqlImage from "../assets/images/sql.png";
+import { Link } from "react-router-dom";
```
Comment (henryjeff) [inline src/routes/About.tsx]: unused imports

File: src/routes/About.tsx [hunk 2]
```diff
@@ -8,60 +8,64 @@ import AwsImage from "../assets/images/aws.png";
...
+import { NONAME } from "dns";
```
Comment (henryjeff) [inline src/routes/About.tsx]: unused imports

File: src/routes/About.tsx [hunk 3]
```diff
@@ -8,60 +8,64 @@ import AwsImage from "../assets/images/aws.png";
...
 
 interface MemberProps {
   name: string;
   role: string;
+  github: string;
```
Comment (henryjeff) [inline src/routes/About.tsx]: generic descriptor? github what? is this a username? email? **_link_**?

**Review by henryjeff — CHANGES_REQUESTED** [review]
Overall decent PR, adds functionality, but sacrifices immensely in code readability and standard practices. Please resolve the issues and resubmit for review.

File: src/routes/About.tsx (hunk 3 above)
Comment (bricct) [reply src/routes/About.tsx]: its actually a github _**suppository**_

File: src/routes/About.tsx [hunk 4]
```diff
@@ -12,56 +12,58 @@ import SqlImage from "../assets/images/sql.png";
 interface MemberProps {
   name: string;
   role: string;
+  linkedInLink: string;
```
Comment (henryjeff) [inline src/routes/About.tsx:15]: good

**Review by henryjeff — APPROVED** [review]

**Pull request merged**